src/ai/backend/manager/models/alembic/versions/565293b7395a_add_quota_scope_id.py

```diff
@@ -0,0 +1,38 @@
+"""add-quota_scope_id
+
+Revision ID: 565293b7395a
+Revises: 10c58e701d87
+Create Date: 2023-03-29 07:29:27.592211
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "565293b7395a"
+down_revision = "10c58e701d87"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("vfolders", sa.Column("quota_scope_id", sa.String(length=64), nullable=True))
+    op.execute(
+        "UPDATE vfolders "
+        "SET quota_scope_id = REPLACE(\"group\"::text, '-', '') "
+        "WHERE ownership_type = 'group';"
+    )
```

Comment:
I remember that raw text statement should be passed by `sqlalchemy.sql.text` method
```suggestion
    from sqlalchemy.sql import text
    op.execute(
        text("UPDATE vfolders "
        "SET quota_scope_id = REPLACE(\"group\"::text, '-', '') "
        "WHERE ownership_type = 'group';"
        )
    )
```